docs/sdk/access-control/solana/overview.md

```diff
@@ -0,0 +1,9 @@
+# Solana Based Access Control
+
```

Comment:
I might just add a sentence at the beginning that quickly summarizes what this guide will be covering. Something like "This guide demonstrates how you can use Lit to permit [decryption](https://developer.litprotocol.com/sdk/access-control/intro) rights to specific addresses on Solana"

Reply:
This page is not specific to any guide, it's regarding ACCs for Solana in general